packages/logger/src/index.ts

```
@@ -0,0 +1,126 @@
+      message,
+      data,
+      error,
+      module: this.getCallerModule(),
+    };
+
+    const formatted = this.formatLog(context);
+
+    // Output based on level
+    switch (level) {
+      case LogLevel.ERROR:
+        console.error(formatted, error || '');
```

ESLint

[error] Unexpected console statement.

(no-console)